<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Asn1Acn::Internal::OptionsPages::ServiceOptionsPage</class>
 <widget class="QWidget" name="Asn1Acn::Internal::OptionsPages::ServiceOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="pathLabel">
     <property name="text">
      <string>Service path</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="uriLabel">
     <property name="text">
      <string>Listening URI</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="stayAliveLabel">
     <property name="text">
      <string>Stay alive period</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="stayAliveSpinBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="suffix">
      <string>ms</string>
     </property>
     <property name="minimum">
      <number>500</number>
     </property>
     <property name="maximum">
      <number>60000</number>
     </property>
     <property name="singleStep">
      <number>500</number>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="stayAliveCheckbox">
     <property name="text">
      <string>Disable watchdog</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" colspan="2">
	   <widget class="Utils::PathChooser" name="pathChooser" native="true" />
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="uriEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header>utils/pathchooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>stayAliveCheckbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>stayAliveSpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
